ansible/templates/common/create_user.sh.j2

```diff
@@ -1,5 +1,5 @@
 {% if doc %}
-Create local user {{ omero_user }}, homedir `{{ omero_user_home_dir }}`::
+Create local user {{ omero_user }}, homedir `{{ omero_user_home_dir }}` (run as root)::
```

Comment:
Note here (and any other place using paths) that the backticks are not appropriate sphinx markup.  Use them doubled up to be verbatim, or with `:file:` to specifically mark up a path.  Single backticks mean something completely different.